frontend-react/src/hooks/UseFileHandler.ts

```diff
@@ -223,18 +233,40 @@ function reducer(
                 payload as RequestCompletePayload
             );
             return { ...state, ...requestCompleteState };
+        case FileHandlerActionType.SCHEMA_SELECTED:
+            const selectedSchemaOption = payload as SchemaOption | null;
+
+            // reset anything related to the file if the selected schema format
```

Comment:
@penny-lischer Updated to only clear file stuff if the format is different from the selected file, and `fileInputResetValue` below should prevent that batch file issue we were running into, but I can't reproduce it locally to check.  Let me know what you think!

Reply:
It's working now but I still think the user experience is wonky and maybe the dropdown to select a schema should be placed above the 'Choose from folder' section. Design question.